pkg/kube/snapshot/snapshot.go

```diff
@@ -104,7 +104,7 @@ type Snapshotter interface {
 	// 'deletionPolicy' is the deletion policy to set on the created VSC
 	// 'snapshotMeta' has metadata of the VolumeSnapshot resource that is source reference for VSC created.
```

Comment:
we are no longer accepting `snapshotMeta` as argument, is this comment still valid?

Reply:
changed.